api/src/main/java/org/openmrs/module/appointments/service/AppointmentsService.java

```diff
@@ -75,5 +76,13 @@
     @Transactional
     @Authorized({VIEW_APPOINTMENTS})
     List<Appointment> search(AppointmentSearchRequest appointmentSearchRequest);
+
+    @Transactional
+    @Authorized({VIEW_APPOINTMENTS})
+    Map<String, List<Appointment>> getAppointmentConflicts(Appointment appointment);
+
+    @Transactional
```

Comment:
Do you need `Transactional`? I hope you don't want any persistence when dealing with conflicts

Reply:
we got some error if we are not using transactional. So we have added it again.

Reply:
ok